insights/util/file_permissions.py

```diff
@@ -79,27 +79,31 @@ def __init__(self, line):
         self.line = line
         r = self._PERMISSIONS_PATTERN.search(self.line)
         if r:
-            (self.perms_owner, self.perms_group, self.perms_other,
+            (self.type, self.perms_owner, self.perms_group, self.perms_other,
```

Comment:
@PaulWay would you mind adding the new `self.type` to the above `Attributes` list in docstring?

Reply:
Good point there @xiangce - will do.